src/simularium/ISimulator.ts

```diff
@@ -46,4 +56,11 @@ export interface ISimulator {
     requestFrameByTime(time: number): void;
     /** request trajectory metadata */
     requestTrajectoryFileInfo(fileName: string): void;
+    /** request available metrics */
+    requestAvailableMetrics(): void;
+    // /** request available plots */
+    requestPlotData(
+        data: Record<string, unknown>,
+        plots: Array<PlotConfig>
+    ): void;
```

Comment:
These docstrings seem a little simple since they're basically the name of the function... Consider either adding more detail or removing?